src/Database/Driver/Postgres.php

```diff
@@ -92,6 +97,24 @@ public function connect(): void
             $dsn = "pgsql:dbname={$config['database']}";
         }
 
+        if ($this->_config['ssl']) {
+            if ($this->_config['ssl_mode']) {
+                $dsn .= ';sslmode=' . $this->_config['ssl_mode'];
+            } else {
+                $dsn .= ';sslmode=' . 'allow';
```

Comment:
```suggestion
                $dsn .= ';sslmode=allow';
```

These could be one string.